srcpkgs/ovmf-ia32/template

```diff
@@ -0,0 +1,118 @@
+# Template file for 'ovmf-ia32'
+pkgname=ovmf-ia32
+version=201911
+revision=1
+archs=noarch
+wrksrc="edk2-edk2-stable${version}"
+hostmakedepends="make python acpica-utils nasm libuuid-devel"
+_basedesc="Tianocore UEFI firmware for qemu"
+short_desc="${_basedesc} ia32"
+maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
+license="BSD-2-Clause-Patent, MIT, Apache-2.0"
+homepage="https://github.com/tianocore/edk2/tree/master/OvmfPkg"
+_openssl_ver=1.1.1d
+distfiles="https://github.com/tianocore/edk2/archive/edk2-stable${version}.tar.gz
+https://www.openssl.org/source/openssl-${_openssl_ver}.tar.gz"
+checksum="23affd4ca2ba526747e72cbb350a4c95d192ac14eeb616778b1976577ed06001
+1e3a91bc1f9dfce01af26026f856e064eab4c8ee0a8f457b5ae30b40b8b711f2"
+_toolchain_opt=GCC5
+
+_ovmf_common_flags="
+ -n ${XBPS_MAKEJOBS}
+ -b RELEASE
+ -D NETWORK_IP6_ENABLE
+ -D TPM2_ENABLE
+"
+_ovmf_x86_flags="
+ ${_ovmf_common_flags}
+ -D HTTP_BOOT_ENABLE
+ -D TLS_ENABLE
+ -D FD_SIZE_2MB
+"
+_ovmf_x86_sb_flags="
+ ${_ovmf_x86_flags}
+ -D SECURE_BOOT_ENABLE
+ -D SMM_REQUIRE
+ -D EXCLUDE_SHELL_FROM_FD
+"
+_ovmf_build_x64=true
+
+case ${XBPS_MACHINE} in
+	i686)
+		# edk2 doesn't find the correct host
+		# also, x64 can't be built on i686
+		export HOST_ARCH=IA32
+		_ovmf_build_x64=false
+		;;
+esac
```

Comment:
Why is `HOST_ARCH` exported on the toplevel, this leaks.

Usually we don't do `_ovmf_build_x64=false`, we either have it set or unset and then test if the variable is set `[ "$foo" ]` instead of making it a command that has to be executed, even if its builtin.